cmd/sync/main.go

```diff
@@ -158,3 +164,35 @@ func main() {
 		}
 	}
 }
+
+func SetMaxConnsDefaultValueIfAbsent(maxConns int, defaultValue int) int {
```

Comment:
A few things on these methods.

1. They should be private (lowercase) to the module
2. I think we can refactor the 4 methods in 2, one for times and another one for ints (for example: `setPositiveIntOrDefault` and `setTimeOrDefault`, to avoid repetition of code. This will also make unit testing easier (less tests).
3. It would be nice to add unit tests to these small methods